src/test/resources/deleteFileSystemItems.feature

```diff
@@ -47,64 +51,97 @@ Feature: FileSystem Delete
+    When user with token "900000" wants to see the content of folder with path "/foo/bar"
     Then response status code is 200
     And the response contains the file with fileSystemId 3 and name "subversion.exe"
-    When user with token "900000" wants to see the content of folder with path "/bla"
+    When user with token "900000" wants to see the content of folder with path "/foo"
     Then response status code is 200
-    And the response contains the folder with fileSystemId 1 and name "fasel"
+    And the response contains the folder with fileSystemId 1 and name "bar"
     And the response does not contains the file with fileSystemId 2 and name "git.exe"
 
   Scenario: insufficient authorization
     Given user with the userId 1234 is allowed to VIEW the fileSystemItem with the fileSystemId 42
     When user with token "900000" wants to delete the fileSystemItem with the fileSystemId 42
     Then response status code is 400
-    And response contains key "message" and value "Folder does not exist, or you are not allowed to edit the folder."
+    And response contains key "message" and value "FileSystemEntity could not be deleted. FileSystemId was 42"
 
   Scenario: insufficient permission
     Given user 9877 exists
     And accessToken with value "2345678" exists for user 9877
     When user with token "2345678" wants to delete the fileSystemItem with the fileSystemId 42
     Then response status code is 400
-    And response contains key "message" and value "Folder does not exist, or you are not allowed to edit the folder."
+    And response contains key "message" and value "FileSystemEntity could not be deleted. FileSystemId was 42"
 
   Scenario: Folder does not exist
     When user with token "900000" wants to delete the fileSystemItem with the fileSystemId 42432567
     Then response status code is 400
-    And response contains key "message" and value "Folder does not exist, or you are not allowed to edit the folder."
+    And response contains key "message" and value "FileSystemEntity could not be deleted. FileSystemId was 42432567"
+
+  Scenario: Folder was created by runtime user.
+    Given database is empty
+    # If this fails check the runtime user id.
+    And fileSystemItem with the fileSystemId 0 exists, was created by user with userId 0 has the path "/" and name "HOME_kevin"
+    And user with userId 123123123 exists and has username "kevin", password "securePassword123"
+    And user with the userId 123123123 is allowed to VIEW the fileSystemItem with the fileSystemId 0
+    And user with the userId 123123123 is allowed to EDIT the fileSystemItem with the fileSystemId 0
+    And accessToken with value "token" exists for user 123123123
+    When user with token "token" wants to delete the fileSystemItem with the fileSystemId 0
+    Then response status code is 400
+    And response contains key "message" and value "FileSystemEntity could not be deleted. FileSystemId was 0"
+
+  Scenario: File was created by runtime user.
```

Comment:
can there even be files created by the runtime user?

Reply:
Maybe in the future.
`ReadMe.txt` or smth.